pkg/controller/audit/actuator.go

```diff
@@ -784,6 +784,15 @@ func seedObjects(auditConfig *v1alpha1.AuditConfig, secrets map[string]*corev1.S
 			}
 		}
 
+		splunkConfigFilter := make(map[string]string, len(auditConfig.Backends.Splunk.CustomData)+2)
+		if len(auditConfig.Backends.Splunk.CustomData) > 0 {
+			splunkConfigFilter["name"] = "modify"
+			splunkConfigFilter["match"] = "*"
```

Comment:
If this is constant anyway, wouldn't it be better to just move this part to the fluentbit config package? So add it to the template like:

```
    match *
    name modify
{{- range $key, $value := $filter }}
    add {{ $key | trim }} {{ $value | trim }}{{ end }}{{ end }}
```

I think it's a bit better because it keeps the fluentbit-specific syntax at a single place.

Reply:
Hm. This would mean that the generic `Filter` field in `fluentbitconfig.Config` can only be used to create a `modify` filter that matches everything. That is sufficient for now, true, but it leaks the custom data business case into the very generic `fluentbitconfig` package. That's why I put all the details into `actuator.go`.

Reply:
I am actually happy with both ways for now. In the long run I would expect the implementation to become easier to maintain when we try to bundle configuration options like these in the fluentbitconfig package. But at this stage of the project it's not necessary. I will accept this PR as is.